x-pack/plugins/security_solution/server/lib/detection_engine/signals/threshold/find_previous_threshold_signals.ts

```diff
@@ -52,6 +51,13 @@ export const findPreviousThresholdSignals = async ({
             'signal.rule.rule_id': ruleId,
           },
         },
+        {
```

Comment:
We might find a signal that was generated on the interval for old data... make sure to exclude those.